Cargo.toml

```diff
@@ -16,6 +16,7 @@ members = [
     "opentelemetry-semantic-conventions",
     "opentelemetry-stackdriver",
     "opentelemetry-stdout",
+    "opentelemetry-userevents-exporter",
```

Comment:
Do we eventually want one or multiple crates to contain exporters to user events? If multiple, maybe rename this to "opentelemetry-user-events-logs"?